tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -69,9 +69,9 @@ stages:
         versionSpec: $(pythonVersion)
 
     - task: JavaToolInstaller@0
-      displayName: Use jdk 11
+      displayName: Use jdk 17
       inputs:
-        versionSpec: '11'
+        versionSpec: '17'
```

Comment:
We have JDK 17 at:
/opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.12-7 and /usr/lib/jvm/temurin-17-jdk-amd64. 

So I am not sure  why this step is not working.

Reply:
I re-run the pipeline, now it passes.